dali/python/setup.py.in

```diff
@@ -33,9 +33,5 @@ setup(name='nvidia-dali@DALI_FLAVOR_MINUS@',
           'console_scripts': [
               'rec2idx = rec2idx:main',
               ],
-          },
-      install_requires = [
-          'future',
```

Comment:
It may fail for python 2.7 (like new print or division).

Reply:
I am trying to figure out whether this is really necessary. This suggests `future` is required for the installation script itself.